### DISCUSSION TITLE
N00B: How does this compare with Nix?

### DISCUSSION THREAD

**Discussion by hacker-DOM** [jdx/mise#2251] [Q&A]
What is the set of same problems they solve, what sets do they each solve uniquely?

I mean Nix as a package managment / dependency management, not necessarily the Os (I use macos).

**Comment by airtonix** [comment]
Answer: In the same way that ASDF differs from nix: 

- not insanely complicated like nix is
- don't have to wait for someone to package a version of a thing like you do with nix
- to use a thing, you don't have to NIX the universe.
